app/views/surveys/_form.html.haml

```diff
@@ -0,0 +1,56 @@
+= form_for @survey, html: { class: "well form-inline", style: "margin-bottom:40px;" } do |f|
+  - if @survey.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@survey.errors.count, "error")} prohibited this survey from being saved:"
+      %ul
+        - @survey.errors.full_messages.each do |message|
+          %li= message
+
+  %h3= t('surveys.new.header')
+
+  .field
+    = f.label :difficulty
+    = render partial: "surveys/star_rating", locals: { |
+            star_5: "rating_difficulty_5",             |
+            star_4: "rating_difficulty_4",             |
+            star_3: "rating_difficulty_3",             |
+            star_2: "rating_difficulty_2",             |
+            star_1: "rating_difficulty_1",             |
+            star_0: "rating_difficulty_0",             |
```

Comment:
The category is passed in as well, is there a reason all of these variables need to be passed in as well? It seems all of this information can be figured out based on the category.